.travis.yml

```diff
@@ -3,10 +3,11 @@ bundler_args: --without debug release
 before_install:
   - gem install bundler
 rvm:
-  - 1.9.3
-  - 2.0.0
+  - 2.1.9
```

Comment:
Yeah I missed it because `.10` is listed way after `.9` on http://rubies.travis-ci.org/

Will fix